server/kit/README.md

```diff
@@ -3,9 +3,8 @@
 * The rationale behind this package:
     * A more opinionated server with fewer choices.
     * go-kit is used for serving HTTP/JSON & gRPC is used for serving HTTP2/RPC
-    * Monitoring and metrics are handled by a sidecar (ie. Cloud Endpoints)
-    * Logs always go to stdout/stderr
+    * Logs always go to stdout/stderr by default, but if running on App Engine, trace enabled Stackdriver logging will be used instead.
     * Using Go's 1.8 graceful HTTP shutdown
-    * Services using this package are meant for deploy to GCP with GKE and Cloud Endpoints.
+    * Services using this package are meant for deploy to GCP.
```

Comment:
for deploy to GCP --> for deploy to GAE ?

Reply:
This is still a viable server for GKE/GCE especially for gRPC services. I just removed the GKE + Cloud Endpoints as we have this fancy new GAE runtime as well.

Reply:
I do hope to add automatic tracing for GKE/GAE next via OpenCensus.

Reply:
Ah, I understand now what you're trying to say. As long as there's a stackdriver env we're good.

👍